tests/settings.py

```diff
@@ -3,6 +3,7 @@
 INSTALLED_APPS = (
     'django.contrib.auth',
     'django.contrib.contenttypes',
+    'django.contrib.admin',
```

Comment:
@e-kolpakov I couldn't get the workbench to load without it. I suspect it's a side effect of having a model-- something upstream which uses the admin interface is getting called because of it.